apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.ts

```diff
@@ -14,4 +14,10 @@ export class CardBarComponent {
 
     /** bar hex color */
     @Input() color = '#1866b4';
+
+    /** Change percentage value */
+    @Input() displayChange: false;
+
+    /** Change value. minimum 0, maximum 100 */
+    @Input() change = 0;
```

Comment:
Is it needed? If so you could you change `displayChange: false` to `displayChange = false`?

Reply:
fixed to displayChange = false